src/app/home/home.component.html

```diff
@@ -21,21 +21,12 @@
 </mat-menu>
 <!--Notification dropdown menu-->
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
-  <span class="menu-item-notification" mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
-    <mat-divider></mat-divider>
-    <p class="menu-item-text" *ngIf="notification.doc.status==='unread' && notification.doc.link">
-      <a href="{{notification.doc.link}}">{{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}</a>
-    </p>
-    <p class="menu-item-text" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">
-      {{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}
-    </p>
-    <p *ngIf="notification.status!=='unread' && notification.doc.link">
-      <a href="{{notification.doc.link}}">{{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}</a>
-    </p>
-    <p *ngIf="notification.status!=='unread' && notification.doc.link===''">
-      {{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}
-    </p>
-  </span>
+  <mat-list *ngFor="let notification of notifications">
+    <mat-list-item (click)="readNotification(notification.doc)" *ngIf="notification.doc.status=='unread'" class="menu-item-text">

```

Comment:
This has been changed in https://github.com/ole-vi/planet/pull/432. So could you please look into the new changes.

Reply:
Yes, please rebase and use version from master